src/ansys/units/variable_descriptor/variable_descriptor.py

```diff
@@ -255,3 +255,10 @@ def add(
 
 for name, dimension in mesh_variables:
     VariableCatalog.add(name, dimension, "mesh")
+
+VariableCatalog.mesh.__doc__ = (
+    "Dictionary of variable descriptors for mesh-related quantities."
+)
+VariableCatalog.fluent.__doc__ = (
+    "Dictionary of variable descriptors for Fluent naming strategy."
+)
```

Comment:
Docstrings were demanded for these. It's not clear if this has worked. Warnings randomly disappear and reappear. (I'm implying that the tool is poor.)

Reply:
Actually if sphinx is able to import this during import of `variable_descriptor.py` then it will generate the docstring without any warning otherwise it will generate the warning. We can check whether docs are rendered properly for this file or not. If it is rendered properly then we don't need to worry about the warning behavior.

Reply:
Sphinx can render a docstring assigned like VariableCatalog.fluent.__doc__ = ... as long as the assignment happens before Sphinx imports the module for autodoc. It's recommended to place the docstring in it's definition directly.